styles.css

```diff
@@ -0,0 +1,91 @@
+h2 {
+  font-family: "Playfair Display SC", serif;
+  font-size: 42px;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+
+  line-height: 5px;
+  color: white;
+}
+nav {
+  font-family: "Open Sans", sans-serif;
+  font-style: italic;
+  font-size: 16px;
+  line-height: 10px;
+  word-spacing: 5px;
+  color: gray;
+}
+a {
+  color: rgb(75, 75, 75);
+}
+a:link {
+  text-decoration: none;
+}
+a:visited {
+  text-decoration: none;
+}
+a:hover {
+  text-decoration: underline;
+}
+.divider{
+  font-style: normal;
+}
+
+.time {
+  color: rgb(75, 75, 75);
+}
+img {
+  border-radius: 25px;
+  float: right;
```

Comment:
Because you have used `float`, you must also clear your floats. See my docs on [using `float`](https://mica-web.github.io/learn/css/float) for more info on this.